resources/public/api/conf/2.0/schemas/uk_property_historic_fhl_period_summary_create_and_amend/request.json

```diff
@@ -0,0 +1,101 @@
+{
+  "title": "request",
+  "description": "Hi there. I'm waiting for the endpoint description to be finalised. Please continue with your review.",
```

Comment:
Is it possible to use the current non-finalised description in the meantime? (Albeit without the "use the XXX endpoint" sentence).  Just seen "temporary" text like this get merged into projects too often....

Reply:
Updated text